libsolidity/interface/ABI.cpp

```diff
@@ -36,7 +36,10 @@ Json::Value ABI::generate(ContractDefinition const& _contractDef)
 		Json::Value method;
 		method["type"] = "function";
 		method["name"] = it.second->declaration().name();
-		method["constant"] = it.second->isConstant();
+		// FIXME: constant should be removed at the next breaking release
+		method["constant"] = it.second->isView();
+		method["view"] = it.second->isView();
+		method["pure"] = it.second->isPure();
```

Comment:
Once we decide on a naming, the ABI should be:
- `constant` (legacy field, to be removed in the future)
- `payable` (legacy field, to be removed in the future)
- `mutability` with values `pure`, `view`, `write`, `payable`